package/network/utils/iproute2/Makefile

```
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iproute2
-PKG_VERSION:=6.0.0
+PKG_VERSION:=6.1.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.xz
```
This fixes the download of the package.